tests/rerankings/test_reranking_ovms.sh

```diff
@@ -69,7 +69,7 @@ function validate_microservice() {
 
 function stop_docker() {
     cd $WORKPATH/comps/rerankings/deployment/docker_compose
-    docker compose -f compose.yaml down ${service_name} --remove-orphans
+    docker compose -f compose.yaml down --remove-orphans
```

Comment:
This only removes `ovms-reranking-server` service but near line 9 of this same file we have:
```
service_name="ovms-reranking-server ovms-reranking-serving"
```
which means 2 services (and it's dependants) are spawned up by this test.

With this change `ovms-reranking-serving` service is left around and not cleaned up.

Reply:
Add `${service_name}` does not work correct with `docker compose -f compose.yaml down`, so we remove it. https://github.com/opea-project/GenAIComps/issues/1323#issuecomment-2748471083